README.md

```diff
@@ -164,21 +164,6 @@ Share ideas on how to improve this project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
```

Comment:
- [OPTIONAL] You have repeated the `Contributing`, `Show your support` , and `Acknowledgments` sections twice from your README file kindly remove one 🚮 

![image](https://user-images.githubusercontent.com/79058364/217681858-fd68db1c-e885-4ae1-947f-e537303dbf97.png)